packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -55,49 +66,94 @@ const PlusCard = ({
   productOption: plan,
   onClickNext,
 }: PlusCardProps): ReactElement => {
+  const id = useId();
+  const { logSubscriptionEvent } = usePlusSubscription();
+  const { earlyAdopterPlanId, productOptions } = usePaymentContext();
+  const pricingIds = useFeature(feature.pricingIds);
+
   const isPaidPlan = !!plan;
+  const cardContentName = isPaidPlan
+    ? OnboardingPlans.Plus
+    : OnboardingPlans.Free;
+  const { heading, features } = cardContent[cardContentName];
+
+  const { hasDiscount, discountPlan } = productOptions.reduce(
+    (acc, product) => {
+      if (!isPaidPlan || !earlyAdopterPlanId) {
+        return acc;
+      }
+
+      const isCardPlan = product.value === plan.value;
+      const isDiscountPlan = product.value === earlyAdopterPlanId;
+      const isMonthly = pricingIds[product.value] === PlusPriceType.Monthly;
+
+      return {
+        hasDiscount: acc.hasDiscount || (isCardPlan && isMonthly),
+        discountPlan: acc.discountPlan || (isDiscountPlan ? product : null),
+      };
+    },
+    {
+      hasDiscount: false,
+      discountPlan: null,
+    },
+  );
+
   const price = {
-    amount: plan?.price ?? '0',
-    cycle: isPaidPlan ? `Billed ${plan?.label}` : 'Free forever',
+    amount: (hasDiscount ? discountPlan.price : plan?.price) ?? '0',
+    cycle: isPaidPlan ? `Billed ${plan?.label?.toLowerCase()}` : 'Free forever',
   };
-  const { logSubscriptionEvent } = usePlusSubscription();
-  const currentPlan = isPaidPlan ? OnboardingPlans.Plus : OnboardingPlans.Free;
-  const { heading, features } = cardContent[currentPlan];
 
   return (
-    <div className="mx-auto w-70 max-w-full rounded-16 border border-border-subtlest-tertiary bg-surface-float p-4">
+    <li
+      aria-labelledby={`${id}-heading`}
+      className="mx-auto w-[21rem] max-w-full rounded-16 border border-border-subtlest-tertiary bg-surface-float p-4"
```

Comment:
Not sure this w-21 is needed, but non blocking

Reply:
Added because of the card size from figma is not related to tab size or other sizes 👀